Added Ehoi use case (#44)
apiobuild/apiobuild.github.io: content/blog/starting-an-online-store-with-chopin-and-instagram.md

---
title: "Starting an Online Store with Chopin and Instagram"
image: /images/blog/social-build-chopin.png
date: 2020-07-20
tags: ["ecommerce", "website", "small-business", "chopin", "google-sheet", "instagram", "facebook", "chopin-use-case", "use-case", "online-shop"]
hide: true
---

> The first step to start an online business is to create a website. This involves getting a domain, hiring developers, creating contents and more. That’s a lot of work and investment before you can even put your products online. Many small business owners turn to social media (mostly [Facebook](https://www.facebook.com/) and [Instagram](https://www.instagram.com/)) to build up their brand. It’s free, easy, and most people åre already familiar with these platforms. **Our online store creator [Chopin](https://telescope.apiobuild.com/app/chopin) shares these benefits and can join forces with Instagram to help you create an online store with your Instagram posts.**

Reviewer: ```suggestion
The first step to start an online business is to create a website. This involves getting a domain, hiring developers, creating contents and more. That’s a lot of work and investment before you can even put your products online. Many small business owners turn to social media (mostly [Facebook](https://www.facebook.com/) and [Instagram](https://www.instagram.com/)) to build up their brand. It’s free, easy, and most people are already familiar with these platforms. **Our online store creator [Chopin](https://telescope.apiobuild.com/app/chopin) shares these benefits and can join forces with Instagram to help you create an online store with your Instagram posts.**
```

🛍️[Read more: why you don\'t need to hire developers now](https://apiobuild.com/blog/build-a-website-for-your-business-what-do-you-need/)

## Why choose Chopin?

[Chopin by apio](https://telescope.apiobuild.com/app/chopin) is a lightweight static online store generator. You can create and manage your store with just google sheets.

- Use the interface you’re familiar - google sheet
- Real-time update for product and order
- No need to upload images, just use URLs
- It’s completely free! (Do we need to say more?)

🛍️[Read more: more awesome features you can add to your Chopin site!](https://apiobuild.com/blog/create-an-online-store-for-free/#introducing-chopin)

🛍️[Read more: follow this guild and create your store today!](https://apiobuild.com/blog/how-to-create-web-store-with-apio/)

> ## Atelier EHOI

Reviewer: ```suggestion
## Use Case: Atelier EHOI
```

> [Atelier EHOI](https://tinyurl.com/Atelier-EHOI) is a boutique bakery in the New York/New Jersey area. They already established presence on [Facebook](https://www.facebook.com/atelierehoi-115683523541293/) and [Instagram](https://www.instagram.com/atelierehoi/). Prior to using Chopin, they were collecting orders through google form which makes the transition painless, as they are already familiar with google products.

Reviewer: ```suggestion
[Atelier EHOI](https://tinyurl.com/Atelier-EHOI) is a boutique bakery in the New York/New Jersey area. They already established presence on [Facebook](https://www.facebook.com/atelierehoi-115683523541293/) and [Instagram](https://www.instagram.com/atelierehoi/). Prior to using Chopin, they were collecting orders through google form which makes the transition seamless, as they are already familiar with google products.
```

[Visit their Chopin site!](tinyurl.com/Atelier-EHOI)

<img src="/images/blog/ehoi-apio.png" class="post-img">

#### Google Sheet and Instagram

With Chopin, all product information including name, price, description, image, etc. is stored in a google sheet. **Atelier EHOI is able to extract URLs from their previous images and posts on Instagram, and use them as product links and images on their Chopin site.** This way, they save time and energy by re-purposing the social content created already and don’t have to craft new content for the online store.

<img src="/images/blog/link-to-ig.png" class="post-img">

#### Real-Time Update

> Atelier EHOI features a weekly updated Dacquoise menu. They announce the secret flavors one week prior to the order deadline. **With Chopin, they can update the cake flavor every week by simply updating a few cells in google sheet.** It not only streamlined the menu update process for store owners, it also made it easier for customers to explore their weekly menu.

Reviewer: ```suggestion
Atelier EHOI features a weekly Dacquoise menu. They announce the secret flavors one week prior to the order deadline. **With Chopin, they can update the cake flavor every week by simply updating a few cells in google sheet.** It not only streamlined the menu update process for store owners, it also made it easier for customers to explore their weekly menu.
```

<img src="/images/blog/update-catalog.png" class="post-img">

#### Announcement

During summer, Atelier EHOI runs a special campaign with a 40% discount. They are able to inform customers with price deduction at the top of their website to boost online sales.

<img src="/images/blog/ehoi-annoucement.png" class="post-img">

#### Order Summary and Shipping Calculation

No more manual calculation. With Chopin, all customers’ orders are summed and applied with free shipping rule. No need to pull out the old calculator anymore!

<img src="/images/blog/order-summary.png" class="post-img">

## Summary

> Atelier EHOI is a perfect example of how simple tool integration can smooth online business operations. The onboarding process took only a couple days. There's no need to learn a monstrous platform or different tools and can plug in Instagram contents to start an online store.

Reviewer: ```suggestion
Atelier EHOI is a perfect example of how simple tool integration can smooth online business operations. The onboarding process took only a couple days. There's no need to learn a monstrous platform or different tools. Just plug in Instagram contents that you've already created to start an online store.
```
